Build failure without removal of this flag.

Needs investigation before filing upstream bug; may be Solaris-specific issue.

--- libcroco-0.6.8/configure.in	2012-10-15 15:55:43.000000000 -0700
+++ libcroco-0.6.8/configure.in	2015-05-22 16:54:40.646534907 -0700
@@ -133,7 +133,7 @@
   -Wstrict-aliasing=2 -Winit-self -Wunsafe-loop-optimizations \
   -Wdeclaration-after-statement -Wold-style-definition \
   -Wno-missing-field-initializers -Wno-unused-parameter \
-  -fno-common -Wno-switch-enum])
+  -Wno-switch-enum])
 
 ##########################################
 
